coverage-plugin/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -4,6 +4,6 @@
 #Fri Dec 22 14:41:56 CET 2023
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```

Comment:
```suggestion
```
I think artifact from other PR

Reply:
Yes I fixed this on another PR. Sorry for confusing. Gradle 8.5 was working locally but than the github build and test jobs failed with the error I mentioned above.